Extensions/TextInput/textinputruntimeobject-pixi-renderer.ts

```diff
@@ -246,8 +271,8 @@ namespace gdjs {
     }
 
     updateOpacity() {
-      if (!this._input) return;
-      this._input.style.opacity = '' + this._object.getOpacity() / 255;
+      if (!this._form) return;
+      this._form.style.opacity = '' + this._object.getOpacity() / 255;
```

Comment:
I think you could use `(this._object.getOpacity() / 255).toFixed(3)` instead. The result of the division could be `0.59200000000005`, and I'm not sure if CSS has some limitations about that.